### PR TITLE
fix: npm publish ignore tests and lambdas sources

### PR DESCRIPTION
**Pull request by sobolk**
#### Description of changes

I've noticed that we ship unnecessary files to npm.
Repro:
- create any npm project `npm init`
- add latest `@aws-amplify/cli-internal` as dependency
- `npm install`
- search for `describe` (or some other test keyword) in node_modules.

See screenshot
image

#### Checklist

- [ ] PR description included
- [ ] `yarn test` passes
- [ ] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/master/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] Any CDK or CloudFormation parameter changes are called out explicitly

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-graphql-model-transformer/.npmignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+**/__mocks__/**
+**/__tests__/**
+src
+publish-notification-lambda
+rds-lambda
+rds-patching-lambda
```
Comment (sobolk) [inline packages/amplify-graphql-model-transformer/.npmignore:6]: ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.

File: packages/amplify-graphql-schema-generator/.npmignore [hunk 2]
```diff
@@ -1,5 +1,6 @@
 **/__mocks__/**
 **/__tests__/**
 src
+vpc-db-lambda
```
Comment (sobolk) [inline packages/amplify-graphql-schema-generator/.npmignore:4]: ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.

File: packages/amplify-graphql-searchable-transformer/.npmignore [hunk 3]
```diff
@@ -0,0 +1,6 @@
+**/__mocks__/**
+**/__tests__/**
+src
+streaming-lambda
```
Comment (sobolk) [inline packages/amplify-graphql-searchable-transformer/.npmignore:4]: ignoring lambdas sources.
it appears that this package users are consuming `zip` files in `lib` not sources.

**Comment by alharris-at** [comment]
Thanks for cutting this PR @sobolk, in order to verify behavior, I'm planning to cut a tagged release, and build into the CLI, then run e2e's there to ensure we're not missing any files we'd otherwise expect.

**Review by alharris-at — APPROVED** [review]

**Pull request merged**